docs/general/whats-new.md

```diff
@@ -1,5 +1,38 @@
 # What's New
 
+## [v2.0.0](https://github.com/project-zot/zot/releases/tag/v2.0.0)
+
+### Updated OCI support
+
+- Support is added for [OCI Distribution Spec v1.1.0-rc3](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc3) and [OCI Image Spec v1.1.0-rc4](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc4).
+
+    :pencil2:  Support is deprecated for earlier release candidates.
+
+### Built-in UI support
+
+- Using the new zot [GUI](../user-guides/user-guide-gui.md), you can browse a zot registry for container images and artifacts. The web interface provides the shell commands for downloading an image using popular third-party tools such as Docker, podman, and skopeo.
+
+### Command line search
```

Comment:
Move this and Bookmarks to the very end of the list

Reply:
Done